app/templates/signup.hbs

```diff
@@ -1,45 +1,26 @@
-{{page-title "Sign Up"}}
-
-<form class="form">
-  <h2>{{this.title}}</h2>
-  {{#each this.fields as |field|}}
-  {{#let this.formData as | myFormData |}}
-    <FormInput
-      @id={{field.id}}
-      @for={{field.for}}
-      @label={{field.label}}
-      @type={{field.type}}
-      @value={{get myFormData field.id}}
-      @placeholder={{field.placeholder}}
-      @errorMessage={{field.errorMessage}}
-      @required={{field.required}}
-      @showError={{field.showError}}
-      @helpMsg={{field.helpMsg}}
-      @onChange={{this.handleFieldChange}}
-      @validator={{field.validator}}
+{{#if this.state}}
+  {{#if (eq this.state "get-started")}}
+    <SignUp::GetStarted
+      @changeRouteParams={{this.changeRouteParams}}
+      @state={{this.state}}
+    />
+  {{else if (eq this.state "thank-you")}}
+    <SignUp::GetStarted
+      @changeRouteParams={{this.changeRouteParams}}
+      @state={{this.state}}
+    />
```

Comment:
What are these if and else doing differently? 🤔

Reply:
I want to render this component if the value of state is either `get-started` or `thank-you`.
But I am not able to find how to use the condition - `if (state = 'get-started' || state = 'thank-you')` in ember. And so I am rendering like this

Reply:
We're already using ember-truth-helpers in our project
Refering this might help: https://www.npmjs.com/package/ember-truth-helpers/v/2.1.0#usage

Reply:
Already tried this. But it's not working.